view/src/pages/home.js

```diff
@@ -0,0 +1,194 @@
+// home.js
+
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import Account from '../components/account';
+import Todo from '../components/todo';
+
+import Drawer from '@material-ui/core/Drawer';
+import AppBar from '@material-ui/core/AppBar';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Toolbar from '@material-ui/core/Toolbar';
+import List from '@material-ui/core/List';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import withStyles from '@material-ui/core/styles/withStyles';
+import AccountBoxIcon from '@material-ui/icons/AccountBox';
+import NotesIcon from '@material-ui/icons/Notes';
+import Avatar from '@material-ui/core/Avatar';
+import ExitToAppIcon from '@material-ui/icons/ExitToApp';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+import { authMiddleWare } from '../util/auth'
+
+const drawerWidth = 240;
+
+const styles = (theme) => ({
+	root: {
```

Comment:
I recommend setting your editor to use spaces rather than tabs, as the width of a tab is not well defined (and you can see that GitHub is rendering them very wide, which makes it difficult to see what's going on).  Google style says to use two spaces, but four is also a common convention.